app/controllers/Whereuat.scala

```diff
@@ -55,24 +40,25 @@ class Whereuat extends Controller {
     (JsPath \ "to").read[String]
   ) tupled
 
-  val atReads : Reads[(String, String, Location)] = (
```

Comment:
I think a short comment explaining this read would be helpful. A comment would help clarify why the `key-location` is nullable.